joblib/numpy_pickle.py

```diff
@@ -414,7 +414,7 @@ def _open_pickle(self, file_handle):
 # Utility functions
 
 def dump(value, filename, compress=0, cache_size=100):
-    """Fast persistence of an arbitrary Python object into a files, with
+    """Fast persistence of an arbitrary Python object into a file, with
```

Comment:
Actually, here it should read 'one or multiple files'. Do you think that you could change that? Thanks!